### DISCUSSION TITLE
What do you know about houston hookups?

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#295] [General]
How do people find each other and just have fun? Without discussing white fence, 5 kids and two labradors on a first date?

**Comment by hernessteve** [comment]
Not much. My friends are using [houston hookups](https://100hookup.com/singles/women/Texas/Reform/) to find dates. It is the most popular website in the past few years in the area. I've been thinking to create there account and never actually did that. Thanks for reminding :)

**Comment by XelaAl** [comment]
I used dating sites only for serious relationships. And I had several great dates.
